vyxal/elements.py

```diff
@@ -4926,7 +4932,7 @@ def uniquify_mask(lhs, ctx):
     """
     lhs = iterable(lhs, ctx=ctx)
     # TODO (user/cgccuser): Reduce code duplication here?
-    if isinstance(lhs, list):
+    if vy_type(lhs, simple=True) == list:
```

Comment:
This will again break it for infinite lazylists. However, instead of going back to the old code, you could use the `def gen()` thing for finite Python `list`s too.